ta4j-examples/src/main/java/ta4jexamples/backtesting/SimpleMovingAverageRangeBacktest.java

```diff
@@ -35,10 +44,8 @@
 import org.ta4j.core.reports.TradingStatement;
 import org.ta4j.core.rules.OverIndicatorRule;
 import org.ta4j.core.rules.UnderIndicatorRule;
-import ta4jexamples.loaders.CsvBarsLoader;
 
-import java.util.ArrayList;
-import java.util.List;
+import ta4jexamples.loaders.CsvBarsLoader;
```

Comment:
Changes are not related and not relevant

Reply:
Those imports are **unnessary**..and instead of saying "Thank you", you reprove me in a unreasonable way. I will not change it. So feel free to close this issue without merging and leave this lib in such a messy way.